stories/core/Table.stories.tsx

```diff
@@ -2292,7 +2295,152 @@ VirtualizedSubRows.argTypes = {
   data: { control: { disable: true } },
 };
 
+export const ColumnManager: Story<Partial<TableProps>> = (args) => {
+  type TableStoryDataType = {
+    index: number;
+    name: string;
+    description: string;
+    id: string;
+    startDate: Date;
+    endDate: Date;
+  };
+
+  const columns = useMemo(
+    (): Column<TableStoryDataType>[] => [
+      {
+        Header: 'Table',
+        columns: [
+          {
+            id: 'index',
+            Header: '#',
+            accessor: 'index',
+            disableToggleVisibility: true,
+          },
+          {
+            id: 'name',
+            Header: 'Name',
+            accessor: 'name',
+          },
+          {
+            id: 'description',
+            Header: 'Description',
+            accessor: 'description',
+            fieldType: 'text',
+          },
+          {
+            id: 'id',
+            Header: 'ID',
+            accessor: 'id',
+          },
+          {
+            id: 'startDate',
+            Header: 'Start date',
+            accessor: 'startDate',
+            Cell: (props: CellProps<TableStoryDataType>) => {
+              return props.row.original.startDate.toLocaleDateString('en-US');
+            },
+          },
+          {
+            id: 'endDate',
+            Header: 'End date',
+            accessor: 'endDate',
+            Cell: (props: CellProps<TableStoryDataType>) => {
+              return props.row.original.endDate.toLocaleDateString('en-US');
+            },
+          },
+        ],
+      },
+    ],
+    [],
+  );
+  const data = useMemo(
+    () => [
+      {
+        index: 1,
+        name: 'Name1',
+        description: 'Description1',
+        id: '111',
+        startDate: new Date('May 1, 2021'),
+        endDate: new Date('Jun 1, 2021'),
+      },
+      {
+        index: 2,
+        name: 'Name2',
+        description: 'Description2',
+        id: '222',
+        startDate: new Date('May 2, 2021'),
+        endDate: new Date('Jun 2, 2021'),
+      },
+      {
+        index: 3,
+        name: 'Name3',
+        description: 'Description3',
+        id: '333',
+        startDate: new Date('May 3, 2021'),
+        endDate: new Date('Jun 3, 2021'),
+      },
+    ],
+    [],
+  );
+
+  return (
+    <Table
+      hasColumnManager={true}
+      isSelectable={true}
+      {...args}
+      columns={columns}
+      data={data}
+      emptyTableContent='No data.'
+    />
+  );
+};
+
+ColumnManager.args = { hasColumnManager: true };
+
 export const DraggableColumns: Story<Partial<TableProps>> = (args) => {
+  const columns = useMemo(
```

Comment:
Can you move this to previous place, so it does not show any changes in this story?
Your story could also go last, by time order..

Reply:
Story has been moved

Reply:
Good, but now could you move `columns` in line 2299 in story `DraggableColumns` to the place as it was before, so it is not shown as a change in your PR? :D

Reply:
Story has been moved again :D